src/dataplane-protocol/src/apis.rs

```diff
@@ -4,16 +4,14 @@
 //! Stores Api Keys supported by the SC.
 //!
 
-use dataplane::derive::Encode;
-use dataplane::derive::Decode;
+use crate::derive::{Decode, Encode};
 
 /// API call from client to SPU
 #[fluvio(encode_discriminant)]
 #[derive(Encode, Decode, PartialEq, Debug, Clone, Copy)]
 #[repr(u16)]
 pub enum AdminPublicApiKey {
```

Comment:
This doesn't belong in the `dataplane-protocol` since this is used by consumer and producer not admin